### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by brunoprietog** [plasmicapp/plasmic#11]
Hello!
I've been trying Plasmic but I couldn't build almost anything. I'm blind and my screen reader didn't read me practically any builder buttons. I don't know if the resulting interfaces are accessible either.
Is there a possibility to improve this please?
Thanks

**Comment by plasmicops** [comment]
@brunoprietog thank you for trying out Plasmic!

Would like to know a bit more about your experience. Is this all in the studio, or did you also had difficulties in the project dashboard / our landing page?

**Comment by brunoprietog** [comment]
Hi, sorry for the late reply. The landing page and the part where the projects are accessible, in fact, I was able to do all that without problems and create a project. The problem happens when entering the project, where there is a panel with many buttons. Practically none of them are labeled and my NVDA screen reader only reads "button" in most cases, without knowing what action the control performs. There are also a couple of images that have no alt text, but I couldn't tell exactly what they are. I simply created a new project, and editing it is where these problems occur.

Thanks for your reply!
